spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -59,6 +60,9 @@
 
 	private final Distribution distribution = new Distribution();
 
+	@NestedConfigurationProperty
```

Comment:
I went w/ nested props merely to keep the startup details out of here. Its not being shared elsewhere - is this a proper use of NCP?